src/components/input/AdminFields.tsx

```diff
@@ -0,0 +1,103 @@
+import { Service } from "io-functions-commons/dist/generated/definitions/Service";
+import { ServiceScopeEnum } from "io-functions-commons/dist/generated/definitions/ServiceScope";
+
+import React, { ChangeEvent, FocusEvent } from "react";
+import { WithNamespaces, withNamespaces } from "react-i18next";
+
+type OwnProps = {
+  service: Service;
+  isApiAdmin: boolean;
+  showError: boolean;
+  errors: Record<string, string>;
+  onChange: (event: ChangeEvent<HTMLSelectElement | HTMLInputElement>) => void;
+  onBlur: (
+    prop: keyof Service
+  ) => (event: FocusEvent<HTMLInputElement>) => void;
+};
+
+type Props = WithNamespaces & OwnProps;
+
+const AdminFields = ({
+  service,
+  onChange,
+  onBlur,
+  isApiAdmin,
+  errors,
+  t
+}: Props) => {
+  const renderFields = () => {
+    const scope = service.service_metadata
+      ? service.service_metadata.scope
+      : undefined;
+    return (
+      <React.Fragment>
+        <div>
+          <label className="m-0">{t("scope")}*</label>
+          <select
```

Comment:
What do you think if we keep the `scope` field where it was before, and in here we render a message (possibly red to keep admin operator attention on it) only when the scope is `NATIONAL`.
The message could be something like `ATTENTION: the current service has the scope value equals to NATIONAL`